azure-quantum/azure/quantum/qiskit/job.py

```diff
@@ -135,41 +137,61 @@ def _format_results(self):
                 job_result["data"] = self._format_unknown_results()
                 job_result["header"] = { "name": self._azure_job.details.name }
 
-        shots = self._azure_job.details.input_params[shots_key] if shots_key in self._azure_job.details.input_params else self._backend.options.get(shots_key)
+        shots = self._azure_job.details.input_params[shots_key] \
+            if shots_key in self._azure_job.details.input_params \
+            else self._backend.options.get(shots_key)
         job_result["shots"] = shots
         return job_result
 
-    def _format_ionq_results(self):
+    @staticmethod
+    def _draw_random_sample(sampler_seed, probabilities, shots):

```

Comment:
is this drawing a random sample? looks more like it's calculating a bunch of values instead of just drawing one?

Reply:
In statistics, a "sample" typically refers to a set of data points, see: https://en.wikipedia.org/wiki/Sample_(statistics)